client.go

```diff
@@ -0,0 +1,75 @@
+package nomadclient
+
+import (
+	"crypto/tls"
+	"crypto/x509"
+	"errors"
+	dphttp "github.com/ONSdigital/dp-net/http"
+	"io/ioutil"
+	"net/http"
+	"strings"
+)
+
+type Nomad struct {
```

Comment:
Again just a short comment on exported variable, `// Nomad is a nomad client which can be used to make requests to the server`